SPECS/podman/podman.spec

```diff
@@ -0,0 +1,3191 @@
+%global with_check 0
```

Comment:
Please remove `with_check` and all of its uses - we control running tests through our tooling (the `RUN_CHECK` argument).